src/admin/migrations/m210605_172811_cms_website_permissions.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m210605_172811_cms_website_permissions
+ */
+class m210605_172811_cms_website_permissions extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn(\luya\cms\models\Website::tableName(), 'group_ids', $this->string());
```

Comment:
please add the class to the use statements. Keep in mind that string contains only 255 chars, even do it could have quit a large number of groups and users, i would suggest using text when working with json